### PR TITLE
Add graded classifier

### PR DESCRIPTION
**Pull request by jkilpatr**

### CODE REVIEW

**Comment by aakarshg** [comment]
Finished both as requested.

File: bml/lib/data_summary.py [hunk 1]
```diff
@@ -33,29 +42,65 @@ def data_summary(data):
...
+                exit(1)
+
+            #print("EEEEEEEEEEENNNNNNNNNNDDDDDD")
+            output_string=output_string+ str(output_prediction) + "\n"
+            #c.execute('INSERT INTO scenario_test VALUES (?,?,?,?,?)',(str(uuid),str(osp_version),str(test_run.scenario_name),str(test_name),average_runtime))
+            '''
+            if test_run.cloud_name in config['clouds']:
+                c.execute('INSERT INTO avg_values2 VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)',\
+                          (str(uuid),str(test_run.timestamp),str(osp_version),str(test_run.cloud_name),\
+                           str(test_run.os_name),str(test_run.kernel),test_run.no_computes,\
```
Comment (jkilpatr) [inline bml/lib/data_summary.py]: you didn't update this.

Comment (jkilpatr) [inline bml/lib/data_summary.py]: it's still no_computes not num_computes this won't run.

Comment (aakarshg) [inline bml/lib/data_summary.py]: This part is actually commented but fixed the variable nomenclature and also ensured that it now displays the std_dev.

**Comment by jkilpatr** [comment]
looking good, lets go.

**Pull request merged**